src/helpers/filter-meeting-data.ts

```diff
@@ -33,15 +33,11 @@ export function filterMeetingData(
         );
       } else {
         //get the union of other filters (Monday OR Tuesday)
-        matchGroups.push(
-          [].concat.apply(
-            [],
-            // @ts-expect-error TODO
-            state.input[filter].map(
-              key => getIndexByKey(state.indexes[filter], key)?.slugs ?? []
-            )
-          )
-        );
+        matchGroups.push([
+          ...state.input[filter]
+            .map(key => getIndexByKey(state.indexes[filter], key)?.slugs ?? [])
+            .flat(),
+        ]);
```

Comment:
had to fix all the `// @ts-expect-error`s for jest - these places are the only places where syntax changes. i tested this of course but if there's a regression it's in one of these places